### PR TITLE
Saxparser

### PR DESCRIPTION
**Pull request by Stanislava88**
In this PR i have solved the task to retrieve data from xml document using sax parser.

### CODE REVIEW

File: tests/com/clouway/saxparser/SaxParserTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class SaxParserTest {
+  Parser saxParser = new SaxParser();
```
Comment (freekman) [inline tests/com/clouway/saxparser/SaxParserTest.java]: public,private ?

File: tests/com/clouway/saxparser/SaxParserTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
```
Comment (freekman) [inline tests/com/clouway/saxparser/SaxParserTest.java]: format

File: src/com/clouway/saxparser/SaxHandler.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+      field.set(object, children);
+    }
+  }
+
+  private void setValue(String qName, Object object) throws SAXException {
+    Field[] fields = object.getClass().getDeclaredFields();
+    for (Field field : fields)
+      try {
+
```
Comment (freekman) [inline src/com/clouway/saxparser/SaxHandler.java]: there is no need to put an empty line here

File: src/com/clouway/saxparser/SaxHandler.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        if (isCollection(field)) {
+          createCollectionObject(qName, field, root);
+        }
+      }
+    }
+  }
+
+  private void createSubChildren(String qName, Object object, List<T> list) throws InstantiationException, IllegalAccessException {
+    Field[] fields;
```
Comment (freekman) [inline src/com/clouway/saxparser/SaxHandler.java]: why not make it like ``` Field[] fields = object.getClass().getDeclaredFields();```

**Comment by freekman** [comment]
LGTM afher fix comments but i want you to squash all the commits when you merge

File: tests/com/clouway/saxparser/SaxParserTest.java (hunk 1 above)
Comment (Stanislava88) [inline tests/com/clouway/saxparser/SaxParserTest.java]: fixed

File: tests/com/clouway/saxparser/SaxParserTest.java (hunk 2 above)
Comment (Stanislava88) [inline tests/com/clouway/saxparser/SaxParserTest.java]: done

File: src/com/clouway/saxparser/SaxHandler.java (hunk 4 above)
Comment (Stanislava88) [inline src/com/clouway/saxparser/SaxHandler.java]: fixed

**Pull request merged**